Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -381,20 +368,15 @@ macro wasmPrologue()
     preserveCalleeSavesUsedByWasm()
     reloadMemoryRegistersFromInstance(wasmInstance, ws0, ws1)
 
-    loadp Wasm::Instance::m_owner[wasmInstance], ws0
-    storep ws0, ThisArgumentOffset[cfr]
-if not JSVALUE64
-    storei CellTag, TagOffset + ThisArgumentOffset[cfr]
-end
-
+    storep wasmInstance, CodeBlock[cfr]
```

Comment:
I kinda think we should do what I discussed above with the two enums in `CallFrame.h`. Otherwise you have to remember that in the wasm calling convention CodeBlock is really instance... which people shouldn't have to do IMO.

Reply:
At the very least we should have a comment in the places where we use `CodeBlock` clarifying that it's referring to the JS calling convention and that we're reusing that same offset.